.devcontainer/Dockerfile

```diff
@@ -1,16 +1,58 @@
-FROM ghcr.io/gofractally/psibase-ubuntu-2004-builder:latest
+FROM ghcr.io/gofractally/psibase-ubuntu-2004-builder:latest as stage1
 
-# Install psibase
-WORKDIR /root
-RUN mkdir psibase
-WORKDIR /root/psibase
+# Install deps
+RUN export DEBIAN_FRONTEND=noninteractive   \
+    && apt-get update                       \
+    && apt-get install -yq                  \
+        wget                                \
+        xz-utils                            \
+        unzip                               \
+        gnupg2                              \
+        curl                                \
+        apt-transport-https                 \
+    && apt-get clean -yq                    \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /opt
+# Prometheus:
+RUN useradd --no-create-home --shell /bin/false prome
+RUN useradd --no-create-home --shell /bin/false node_exporter
+COPY prometheus/prometheus.yml /etc/prometheus/prometheus.yml
+ARG PROMVERSION=prometheus-2.43.0+stringlabels.linux-amd64
+RUN mkdir -p /var/lib/prometheus    \
+    && cd /var/lib/prometheus       \
+    && wget https://github.com/prometheus/prometheus/releases/download/v2.43.0%2Bstringlabels/${PROMVERSION}.tar.gz \
+    && tar xf ${PROMVERSION}.tar.gz          \
+    && rm ${PROMVERSION}.tar.gz              \
+    && cp ${PROMVERSION}/prometheus /usr/local/bin/ \
+    && cp ${PROMVERSION}/promtool /usr/local/bin/ \
+    && chown prome:prome /usr/local/bin/prometheus \
+    && chown prome:prome /usr/local/bin/promtool \
+    && cp -r ${PROMVERSION}/consoles /etc/prometheus \
+    && cp -r ${PROMVERSION}/console_libraries /etc/prometheus \
+    && chown -R prome:prome /etc/prometheus/consoles \
+    && chown -R prome:prome /etc/prometheus/console_libraries
 
-# Clone the specified github repo
-RUN git clone https://github.com/gofractally/psibase.git .
+# Tools (websocat & grok) to help export data to prometheus
+RUN cd /root \
+ && wget https://github.com/fstab/grok_exporter/releases/download/v1.0.0.RC5/grok_exporter-1.0.0.RC5.linux-amd64.zip \
+ && unzip grok_exporter-*.zip \
+ && rm grok_exporter-*.zip \
+ && ln -s /root/grok_exporter-1.0.0.RC5.linux-amd64/grok_exporter /usr/local/bin/grok_exporter \
+ && wget https://github.com/vi/websocat/releases/download/v1.11.0/websocat.aarch64-unknown-linux-musl \

```

Comment:
This doesn't seem right. aarch64 is ARM.

Reply:
Weird, now I'm wondering how it was working...

Reply:
(Also, fixed)